my-app/src/thumbNails.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+function ThumbNails(props){
+    return (
+        <div> 
+            <figure>
+                <a href ={props.nextLink}>
+                    <img className = {props.classId} src = {props.filePath} alt="" width="250" height="600"/>
+                    <figcaption>{props.caption}</figcaption>
+                </a>
+            </figure> 
+        </div>
+    );
+}
+export default ThumbNails;
```

Comment:
just remembered this one, instead of constantly doing "props.whatever" we can create a new variable for each one of the variables using JS magic then use those directly.

```suggestion
import React from 'react'

function ThumbNails(props){
    var {nextLink, classId, filePath, caption} = props;
    
    return (
        <div> 
            <figure>
                <a href ={nextLink}>
                    <img className = {classId} src = {filePath} alt="" width="250" height="600"/>
                    <figcaption>{caption}</figcaption>
                </a>
            </figure> 
        </div>
    );
}
export default ThumbNails;
```